types/iconLabel.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
The `IconLabelPair` type should live closer to the overview row components (maybe in `labels.tsx`?) The top-level `types` directory is for types that we expect to use all over the codebase.